cmd/config/subcommand/sandbox/sandbox_config.go

```diff
@@ -18,4 +20,8 @@ type Config struct {
 	// Flyte compliant sandbox image. Usually useful, if you want to push the image to your own registry and relaunch
 	// from there.
 	Image string `json:"image" pflag:",Optional. Provide a fully qualified path to a Flyte compliant docker image."`
+
+	// Optionally it is possible to use local sandbox image
+	// If local flag pass then flytectl will not pull image from registry. Usually useful, if you want to test your local images without pushing them to a registry
+	Local bool `json:"local" pflag:",Optional. Enable if you want to use local available image."`
```

Comment:
I would rather have the option be called ``ImagePullPolicy`` since this may be needed regardless of whether the image is built locally or not... it could be due to auth, somebody will pull the image separately with special authentication, then call ``flytectl sandbox start --imagePullPolicy Never`` to just run it...

```suggestion
	ImagePullPolicy ImagePullPolicy `json:"imagePullPolicy" pflag:",Optional.Defines the image pull behavior."`
```

with
```golang
//go:generate enumer -type=ImagePullPolicy -trimprefix=ImagePullPolicy

type ImagePullPolicy int
const(
    ImagePullPolicyAlways ImagePullPolicy = iota
    ImagePullPolicyIfNotPresent
    ImagePullPolicyNever
)
```

Reply:
@EngHabu enumer works fine but pflag is not working with enumer. Got this

```
INFO[0000] [image] is of a basic type with default value [*new(string)]. 
Error: invalid type. it must be struct, received [int] for field [imagePullPolicy]
Usage:
  pflags MyStructName --package myproject/mypackage [flags]
  pflags [command]

Examples:

// go:generate pflags MyStruct
type MyStruct struct {
	BoolValue        bool              `json:"bl" pflag:"true"`
	NestedType       NestedType        `json:"nested"`
	IntArray         []int             `json:"ints" pflag:"[]int{12%2C1}"`
}
	

Available Commands:
              
  help        Help about any command

Flags:
      --bind-default-var     The generated PFlags Set will bind fields to the default variable.
      --default-var string   Points to a variable to use to load default configs. If specified & found, it'll be used instead of the values specified in the tag. (default "defaultConfig")
  -h, --help                 help for pflags
  -p, --package string       Determines the source/destination package. (default ".")

Use "pflags [command] --help" for more information about a command.

2021/11/16 13:22:05 invalid type. it must be struct, received [int] for field [imagePullPolicy]
sandbox_config.go:3: running "pflags": exit status 1
```